addons/project/views/project_views.xml

```diff
@@ -940,18 +952,27 @@
                                     <field name="legend_normal" invisible="1"/>
                                     <field name="legend_done" invisible="1"/>
                                     <field name="legend_blocked" invisible="1"/>
+                                    <field name="message_needaction" invisible="1" readonly="1"/>
                                     <field name="project_id" invisible="1"/>
-                                    <field name="is_closed" invisible="1"/>
-                                    <field name="sequence" widget="handle"/>
-                                    <field name="name"/>
+                                    <field name="is_closed" invisible="1" />
+                                    <field name="allow_subtasks" invisible="1" />
+                                    <field name="sequence" invisible="1" readonly="1"/>
+                                    <field name="priority" widget="priority" optional="show" nolabel="1"/>
+                                    <field name="name" widget="name_with_subtask_count"/>
                                     <field name="display_project_id" string="Project" optional="hide"/>
+                                    <field name="child_text" invisible="1"/>
                                     <field name="partner_id" optional="hide"/>
+                                    <field name="parent_id" optional="hide" attrs="{'invisible': [('allow_subtasks', '=', False)]}" groups="base.group_no_one"/>
```

Comment:
Don't add this field because the parent one is the one displayed in the form view, so it is useless. 🙂